plugins/org.locationtech.udig.libs/log4j.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
I guess its required to add log4j.xml to build.properties (bin.includes) otherwise it might not be present in final jar.

Reply:
Added.